conans/model/new_build_info.py

```diff
@@ -182,6 +182,8 @@ def aggregated_components(self):
                         current_values = result.get_init("requires", [])
                         current_values.extend(component.requires)
 
+                # We copy the properties from the root object, even if we have components
+                result._generator_properties = copy.copy(self._generator_properties)
```

Comment:
Could this be breaking? I think it shouldn't, the aggregated_components is used only by the new generators, and it makes sense that aggregating the components in a single cppinfo, keeps the root properties. The aggregation cannot be done for the component's properties because they cannot be aggregated but keeping the root one make sense. right?

Reply:
yes.